jina/enums.py

```diff
@@ -240,6 +240,8 @@ class RequestType(BetterEnum):
     SEARCH = 1
     TRAIN = 2
     CONTROL = 3
+    DELETE = 4
```

Comment:
can we use it to put TRAIN as the last one; just in case we eventually remove it we may give less problems

Reply:
okay, I changed it